src/config.ts

```diff
@@ -0,0 +1,185 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import { parseAllowedChannels } from './csv-parser-allowed-channels.js'
+import path from 'path'
+
+export interface DiscordServer {
+  discordToken: string | undefined;
+  discordClientId: string | undefined;
+  discordGuildId: string | undefined;
+  botLanguage: string;
+  discordAnnouncementsChannel: string | undefined;
+  scheduledDiscordEventsEnabled: boolean;
+}
+
+export interface MappedStatusCommands {
+  'pr-request-review': string;
+  'pr-request-changes': string;
+  'pr-approved-by-code-review': string;
+  'pr-task-cancelled': string;
+  'pr-work-in-progress': string;
+  'pr-merged-on-staging': string;
+  'pr-merged-in-prod': string;
+  'pr-done': string;
+}
+
+export interface PrTemplate {
+  allowedChannels: string[];
+}
+
+export interface TimeZone {
+  name: string;
+  value: string;
+}
+
+export interface QAMention {
+  discordQARoleId: string | undefined;
+  discordQAChannelName: string | undefined;
+}
+
+export interface CronScheduleReview {
+  scheduleReview: string;
+  timeZone: string;
+}
+
+export interface RequestPoint {
+  discordAdminPointRequestChannel: string | undefined;
+  discordAdminTagId: string | undefined;
+}
+
+export interface ScheduleMessages {
+  timeZone: string | undefined;
+  pathMarkdownFolder: string;
+}
+
+export interface ScheduleCalendar {
+  scheduledCalendarInterval: string;
+  timeZone: string;
+}
+
+export interface VotePoints {
+  ANSWERS: {
+    text: string;
+    emoji: string;
+  }[];
+  TAG_IDS: {
+    taskCompletedTagId: string;
+    addPointTagId: string;
+    boostedPointTagId: string;
+  };
+}
+
+export interface GeminiIntegration {
+  scheduledGeminiEnabled: boolean;
+  geminiSecret: string | undefined;
+  scheduleTime: string | undefined;
+  interactionsPrompts: string[] | undefined;
+  interactionChannel: string | undefined;
+}
```

Comment:
They are a lot! maybe you can move them to a .d.ts file and import them here instead, so we maintain small files